src/spec.json

```diff
@@ -36,7 +37,8 @@
         },
         {
             "id": 2,
-            "title": "Development",
+            "title": "Business Development",
```

Comment:
@jmstach is this change okay? Here I thought `Development` was too broad and narrowed the scope cuz the questions seemed to relate to business acquisition.